tests/integration/targets/pacman/tasks/package_urls.yml

```
@@ -19,7 +23,14 @@
     - name: Get URL for {{url_pkg}}
       command:
         cmd: pacman --sync --print-format "%l" {{url_pkg}}
-      register: url_pkg_url
+      register: url_pkg_stdout
+    - name: Download {{url_pkg}} pkg
+      get_url:
+        url: '{{url_pkg_stdout.stdout}}'
+        dest: '{{url_pkg_path}}'
+    - name: Host {{url_pkg}}
+      shell:
+        cmd: 'python -m http.server --directory /tmp/ {{http_port}} &'
```

the child python process dies everytime it tries to write to `stderr` because it receives a SIGPIPE. wiring it to /dev/null avoids the problem.

I added a loop to make sure the server is up before continuing.

```suggestion
        cmd: 'python -m http.server --directory {{url_pkg_basepath}} {{port}} >/dev/null 2>&1 &'

    - name: Wait for http.server ({{url_pkg_url}})
      shell:
        cmd: 'while ! curl -v {{url_pkg_url}} >/dev/null; do echo http server not ready, sleeping; sleep 0.2; done;'
```